packages/create-vite/template-qwik/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "npm run build; npx serve dist"
```

Comment:
vite preview fails if you don't call build in all other cases so that part is unneded. And I think we should only have `serve dist` here and add `serve` as a dependency.